test/contracts/HCElection.aes

```diff
@@ -102,12 +104,12 @@ main contract HCElection =
     require(Call.caller == state.leader, "Must be called by the last leader of epoch")
     put(state{pin = Some(proof)})
 
+  payable stateful entrypoint add_reward(height : int, to : address) =
```

Comment:
is there a reason to assert that the height is not in the past? Would not be catastrophic in itself if consensus puts an award in validator pocket, but can someone else fiddle with a small reward and destroy the logic of the contract?

Reply:
Since the reward delay (how many blocks in the past we distribute rewards for) is configurable we would need to carry that value in here to make a correct check. Perhaps we shouldn't let it be configurable (keep it at 1?), for HC, then we can tighten this.

There is another assertion missing here though, this should only be allowed to be called by the consensus logic... Adding that in a later commit.